src/doc/contrib/src/tests/writing.md

```diff
@@ -154,6 +154,48 @@ If you need to test with registry dependencies, see
 If you need to test git dependencies, see [`support::git`] to create a git
 dependency.
 
+#### Cross compilation
```

Comment:
This is currently under “Functional Tests” paragraph. Would we write a UI test for cross-compilation in the future?

Reply:
Yes. It's not clear to me why it was organized this way since everything (except for the code example) also applies to using the template-style tests.